utils/build-presets.ini

```diff
@@ -1024,6 +1024,23 @@ skip-build-swift
 skip-build-cmark
 llvm-include-tests
 
+
+[preset: buildbot_incremental_linux,tsan-libdispatch-test]
+mixin-preset=buildbot_incremental_linux,llvm-only
+build-subdir=buildbot_incremental_tsan_libdispatch_test
+
+llvm-cmake-options=-DLLVM_ENABLE_PROJECTS=clang;libcxx;compiler-rt  \
```

Comment:
This shouldn't be needed. Can't you just re-compile compiler-rt in the python code? It doesn't take a long time to compile compiler-rt...